AppDeployToolkit/AppDeployToolkitConfig.xml

```diff
@@ -460,946 +543,969 @@
+			In alternativa, salvare il lavoro e fare clic su "Chiudi Programmi".</ClosePrompt_Message>
+		<ClosePrompt_CountdownMessage>NOTA: il programma(s) sarà chiuso automaticamente in:</ClosePrompt_CountdownMessage>
+		<DeferPrompt_WelcomeMessage>La seguente applicazione sta per essere installata:</DeferPrompt_WelcomeMessage>
+		<DeferPrompt_ExpiryMessage>Si può decidere di posticipare l'installazione fino alla prossima richiesta automatica:</DeferPrompt_ExpiryMessage>
+		<DeferPrompt_WarningMessage>Una volta che le richieste rimanenti saranno scadute, non sarà più possibile posticipare l'installazione.</DeferPrompt_WarningMessage>
+		<DeferPrompt_RemainingDeferrals>Posticipi rimanenti:</DeferPrompt_RemainingDeferrals>
+		<DeferPrompt_Deadline>Scadenza:</DeferPrompt_Deadline>
+		<WelcomePrompt_CountdownMessage>Il {0} continuerà automaticamente in:</WelcomePrompt_CountdownMessage>
+		<WelcomePrompt_CustomMessage></WelcomePrompt_CustomMessage>
+
+		<DeploymentType_Install>Installazione</DeploymentType_Install>
+		<DeploymentType_UnInstall>Disinstallazione</DeploymentType_UnInstall>
+		<DeploymentType_Repair>Riparazione</DeploymentType_Repair>
+		<BalloonText_Start>Iniziata.</BalloonText_Start>
+		<BalloonText_Complete>Completata.</BalloonText_Complete>
+		<BalloonText_RestartRequired>Completata. È necessario riavviare il computer.</BalloonText_RestartRequired>
+		<BalloonText_Error>Fallita.</BalloonText_Error>
+		<BalloonText_FastRetry>Non completata.</BalloonText_FastRetry>
+		<Progress_MessageInstall>Installazione in corso. Attendere prego...</Progress_MessageInstall>
+		<Progress_MessageUninstall>Disinstallazione in corso. Attendere prego...</Progress_MessageUninstall>
+		<Progress_MessageRepair>Riparazione in corso. Attendere prego...</Progress_MessageRepair>
+		<BlockExecution_Message>L'esecuzione di questa applicazione è stata temporaneamente bloccata in modo che l'operazione di installazione possa essere completata.</BlockExecution_Message>
+		<RestartPrompt_Title>Riavvio Richiesto</RestartPrompt_Title>
+		<RestartPrompt_Message>Per completare l'installazione, è necessario riavviare il computer.</RestartPrompt_Message>
+		<RestartPrompt_MessageTime>Salvare il lavoro e riavviare entro il tempo assegnato.</RestartPrompt_MessageTime>
+		<RestartPrompt_MessageRestart>Il computer verrà riavviato automaticamente al termine del conto alla rovescia.</RestartPrompt_MessageRestart>
+		<RestartPrompt_TimeRemaining>Tempo rimanente:</RestartPrompt_TimeRemaining>
+		<RestartPrompt_ButtonRestartLater>Minimizzare</RestartPrompt_ButtonRestartLater>
+		<RestartPrompt_ButtonRestartNow>Riavvia Ora</RestartPrompt_ButtonRestartNow>
+
+		<DeploymentType_InstallVerb>installato</DeploymentType_InstallVerb>
+		<DeploymentType_UnInstallVerb>disinstallato</DeploymentType_UnInstallVerb>
+		<DeploymentType_RepairVerb>riparato</DeploymentType_RepairVerb>
+		<NxtWelcomePrompt_FollowApplication>La seguente applicazione sta per essere {0}:</NxtWelcomePrompt_FollowApplication>
```

Comment:
Move all the translations to a different issue, branch and PR, only en and de in this PR